README.md

```diff
@@ -24,13 +24,8 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md) for how to contribute.
 # Status
 
 ### Kafka broker versions with which this version of Afkak is compatible:
-- 0.8.0 *
-- 0.8.1 *
-- 0.8.1.1 *
-- 0.8.2.1
 - 0.9.0.1
-
-\* See [Errata, below](#errata).
+- 1.1.1
```

Comment:
Wat?   Only compatible with 1.1.1?

Reply:
I many run the tests against some older versions once coordinated consumer support is in, but for now this list only includes what we test against.

Reply:
But that's not what it *says*: "### Kafka broker versions with which this version of Afkak is compatible:"

Reply:
Oh, good point... I will change the wording. My intent here is to drop support for brokers below 0.9.0.1.